schemas/location/location.yaml

```diff
@@ -5,4 +5,5 @@ properties:
     description: The applicant's ip address.
   country_of_residence:
     type: string
+    pattern: '^[A-Z]{3}$'
     description: The applicant's country of residence.
```

Comment:
```suggestion
    description: The applicant's country of residence in alpha-3 code.
```

Reply:
Or using the same description as in [here](https://github.com/onfido/onfido-openapi-spec/pull/41/files#diff-9f1d964320d002b42cfd4d9b631aecdcd64d5410107c41939f1ddaf7953c929dR12).